.github/workflows/ci.yaml

```diff
@@ -5,6 +5,9 @@ on: [push, pull_request, workflow_dispatch]
 env:
   DEFAULT_GO_VERSION: ^1.16
   IS_PUSH: ${{ github.event_name == 'push' }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
can we move all the secrets to only be accessible in the specific steps that need them, like [here](https://github.com/aws/aws-node-termination-handler/blob/main/.github/workflows/ci.yml#L134-L138)?